docs/SEEEDSTUDIO_WIO_E5.md

```diff
@@ -117,7 +117,8 @@ The first time that you flash the mLRS firmware to either the E5 Mini or E5 Grov
 ### Subsequent Flashing ###
 
 Subsequent flashing can be done in two ways:
-1. Follow the initial flashing process
+1. Follow the initial flashing process using the ST-Link programmer
     - There is no need to change the read out protection after the initial flash
-2. Boot into the system bootloader using the cli, lua or the OLED display
-    - This enables programming over UART1 (PA9, PA10) or UART2 (PA2, PA3) with a USB-TTL adapter using the Serial connection option in STM32CubeProgrammer
+    - If a full erase is desired, go to the Erase & Programming page and click 'Full chip erase'
+2. Boot into the system bootloader using the CLI, LUA script or the OLED display
```

Comment:
Minor clarification for me.  Makes sense that CLI and OLED are all capitals because they are acronyms but should Lua be stylized the same as the logo like we do for EdgeTX, CRSF, etc?

<img width="315" alt="image" src="https://github.com/olliw42/mLRS-docu/assets/41841496/1bb21719-97b9-4509-a16a-46a9583de5d3">

Reply:
absolutely!! I think I was arguing once this way too LOL LOL